### PR TITLE
is it okay now ? ( Bengali Facts)

### PR DESCRIPTION
**Pull request by dshaw0004**
I run the `npm run test:lint:fix` on node v16. Hope it is fixed now.

### CODE REVIEW

**Comment by dshaw0004** [comment]
even after running `npm run test:lint:fix` it was not fixed so I have to copy the whitespace from `eng-us.js` then paste it to `ben-in.js`.

File: replit.nix [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ pkgs }: {
```
Comment (BeauBouchard) [inline replit.nix:1]: lets just remove this file, and the other `.replit` file

**Review by BeauBouchard — CHANGES_REQUESTED** [review]
I think this is an excellent contribution, Thank you for making changes in a new branch, it makes it a lot cleaner. 

This would be fine to merge if it didn't include the `replit.nix` and `.replit` files, you can remove them by deleting them, and `git add -A` then making a new commit / push. 

Once again thank you for your continued contribution and efforts on this localization.

**Comment by dshaw0004** [comment]
okay

**Comment by dshaw0004** [comment]
its done. sorry I didn't noticed that.

**Review by BeauBouchard — APPROVED** [review]
Great job!

**Pull request merged**